x/xvalidator/validator.go

```diff
@@ -35,3 +36,8 @@ func IsPortMapping(fl validator.FieldLevel) bool {
 	}
 	return true
 }
+
+// IsEnv validates if given field is valid env variable declaration.
+func IsEnv(fl validator.FieldLevel) bool {
```

Comment:
It will be good to check env name (for special chars etc) 

https://stackoverflow.com/questions/2821043/allowed-characters-in-linux-environment-variable-names